test/e2e/page-objects/pages/test-dapp.ts

```diff
@@ -250,6 +250,21 @@ class TestDapp {
     this.driver = driver;
   }
 
+  private readonly networkSelector = (networkId: string) => ({
+    css: '[id="chainId"]',
```

Comment:
just as info, you could use this instead:

```suggestion
    testId: 'chainId',
```